.gitignore

```diff
@@ -1,4 +1,3 @@
 vendor
 launch.json
-echo
-enrober
```

Comment:
It was an artifact from when I had a sub folder named `enrober`. I was just manually ignoring the binary. I'll update the .gitignore.